pkg/cmd/server/server.go

```diff
@@ -289,15 +289,10 @@ func (s *server) run() error {
 	return nil
 }
 
-func getDefaultVolumeSnapshotLocations(arkClient clientset.Interface, namespace string, defaultVolumeSnapshotLocations map[string]string) (map[string]*api.VolumeSnapshotLocation, error) {
-	providerDefaults := make(map[string]*api.VolumeSnapshotLocation)
-	if len(defaultVolumeSnapshotLocations) == 0 {
-		return providerDefaults, nil
-	}
-
+func getVolumeSnapshotLocations(arkClient clientset.Interface, namespace string, defaultVolumeSnapshotLocations map[string]string) (map[string]*api.VolumeSnapshotLocation, error) {
```

Comment:
Why the name change? It's still only getting the default, whether it's because it's the only one or one was defined.

Reply:
@nrb  VSLs don't have a preset default so if the user doesn't specify one, there will be none.

This makes it so in some cases what is returned is the default, and some cases not. This new name is more generic and less misleading, that was my thinking. Makes sense?

Reply:
I don't think I follow - the only time it's used in code outside of tests, it's for default locations. Have you looked at Steve's alternative suggestion?

Reply:
To clarify: I don't think as the default as "the only one", I think of it as the feature that we currently allow to be set only via that `ark server` flag. This means the user doesn't have to specify the name of the VSL location for that provider. 

This is true for BSL. Is it incorrect for VSL?